{"1.0.0":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":["Framework :: Django","License :: Public Domain","Operating System :: OS Independent","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":"django","license":null,"maintainer":null,"maintainer_email":null,"name":"django-command-exception","package_url":"https://pypi.org/project/django-command-exception/","platform":null,"project_url":"https://pypi.org/project/django-command-exception/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/django-command-exception/1.0.0/","requires_dist":["Django","setuptools"],"requires_python":null,"summary":"Django python exception model, admin, logging handler, middleware and excepthook","version":"1.0.0","yanked":false,"yanked_reason":null},"last_serial":24219920,"urls":[{"comment_text":"","digests":{"blake2b_256":"98334896829920fc6261a61004788f3f16a2aac1967f20a707bdc702f9a51f12","md5":"2517da79c1d15a9c4a383b286d8e5381","sha256":"055673638675d625b18a408ae547d20e948a438f45e3476f7f1125f796778143"},"downloads":-1,"filename":"django_command_exception-1.0.0.tar.gz","has_sig":false,"md5_digest":"2517da79c1d15a9c4a383b286d8e5381","packagetype":"sdist","python_version":"source","requires_python":null,"size":2023,"upload_time":"2024-06-28T08:51:11","upload_time_iso_8601":"2024-06-28T08:51:11.044949Z","url":"https://files.pythonhosted.org/packages/98/33/4896829920fc6261a61004788f3f16a2aac1967f20a707bdc702f9a51f12/django_command_exception-1.0.0.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"1.0.1":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":["Framework :: Django","License :: Public Domain","Operating System :: OS Independent","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":"django","license":null,"maintainer":null,"maintainer_email":null,"name":"django-command-exception","package_url":"https://pypi.org/project/django-command-exception/","platform":null,"project_url":"https://pypi.org/project/django-command-exception/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/django-command-exception/1.0.1/","requires_dist":["Django","setuptools"],"requires_python":null,"summary":"Django python exception model, admin, logging handler, middleware and excepthook","version":"1.0.1","yanked":false,"yanked_reason":null},"last_serial":24219920,"urls":[{"comment_text":"","digests":{"blake2b_256":"d1722fc5b08288f0fbdf8c5a030b97f696c8972a69eb0e25faa0d86595e50f03","md5":"ae739b0830b89475f1831e8d30ee48cd","sha256":"6d4f5e3b16b516cedcd4f90dfc84bdc5bf481253f0bcc97ca3ebacc885ffe6f1"},"downloads":-1,"filename":"django_command_exception-1.0.1.tar.gz","has_sig":false,"md5_digest":"ae739b0830b89475f1831e8d30ee48cd","packagetype":"sdist","python_version":"source","requires_python":null,"size":2454,"upload_time":"2024-07-20T15:10:14","upload_time_iso_8601":"2024-07-20T15:10:14.073928Z","url":"https://files.pythonhosted.org/packages/d1/72/2fc5b08288f0fbdf8c5a030b97f696c8972a69eb0e25faa0d86595e50f03/django_command_exception-1.0.1.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"1.0.2":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":["Framework :: Django","License :: Public Domain","Operating System :: OS Independent","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":"django","license":null,"maintainer":null,"maintainer_email":null,"name":"django-command-exception","package_url":"https://pypi.org/project/django-command-exception/","platform":null,"project_url":"https://pypi.org/project/django-command-exception/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/django-command-exception/1.0.2/","requires_dist":["Django","setuptools"],"requires_python":null,"summary":"Django python exception model, admin, logging handler, middleware and excepthook","version":"1.0.2","yanked":false,"yanked_reason":null},"last_serial":24219920,"urls":[{"comment_text":"","digests":{"blake2b_256":"ac51c5296b4dac2478b778987392e892ebaa77dd035dfa838cf4aeaf92e45a7f","md5":"cf2192d771dfa21835f308062bf93d1d","sha256":"c4f0bdc40ac60fc64b0ea0b5122e0af5119791de4c3929a2a4619fcafbffcf52"},"downloads":-1,"filename":"django_command_exception-1.0.2.tar.gz","has_sig":false,"md5_digest":"cf2192d771dfa21835f308062bf93d1d","packagetype":"sdist","python_version":"source","requires_python":null,"size":2646,"upload_time":"2024-07-20T19:58:52","upload_time_iso_8601":"2024-07-20T19:58:52.785282Z","url":"https://files.pythonhosted.org/packages/ac/51/c5296b4dac2478b778987392e892ebaa77dd035dfa838cf4aeaf92e45a7f/django_command_exception-1.0.2.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]}}